src/snowflake/snowpark/_internal/analyzer/expression.py

```diff
@@ -225,6 +225,46 @@ def __init__(self, value: Any, datatype: Optional[DataType] = None) -> None:
             self.datatype = infer_type(value)
 
 
+class Interval(Expression):
+    def __init__(
+        self,
+        year: Optional[int] = 0,
+        quarter: Optional[int] = 0,
+        month: Optional[int] = 0,
+        week: Optional[int] = 0,
+        day: Optional[int] = 0,
+        hour: Optional[int] = 0,
+        minute: Optional[int] = 0,
+        second: Optional[int] = 0,
+        millisecond: Optional[int] = 0,
+        microsecond: Optional[int] = 0,
+        nanosecond: Optional[int] = 0,
```

Comment:
Are these really optional? If they are set to None will your sql still be valid?

Reply:
Oops you're right, let me remove the Optional. The values cannot be None.